packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -222,11 +228,27 @@
 
         &-item {
             display: grid;
-            grid-template-columns: 1fr 1fr 1fr 1fr;
+            @include desktop {
+                grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
can we use `repeat()` which comes along with `grid-template-columns`?
here is the ref: [https://developer.mozilla.org/en-US/docs/Web/CSS/repeat](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat)

@maryia-matskevich-deriv let me know if it works 🙇

Reply:
improved ✅ it works, thank you